src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total = 0;
+
+  if (days < 3) {
+    total = days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    total = days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    total = days * 40 - 50;
+  }
```

Comment:
Reverse order of these `if`s. Than won't need to check the second condition here `days >= 3 && days < 7`